kf/kf_unix.go

```diff
@@ -115,6 +115,43 @@ func VerifyNMountBPFFS() error {
 			return fmt.Errorf("unable to mount %s at %s: %s", srcPath, dstPath, err)
 		}
 	}
+
+	return VerifyNMountTraceFS()
+}
+
+// VerifyNMounTraceFS - Mounting trace filesystem
+func VerifyNMountTraceFS() error {
+	dstPath := "/sys/kernel/tracing"
```

Comment:
https://lwn.net/Articles/630526/
So it appears that tracefs only mount is preferred if debugfs is not already mounted (due to security considerations). So I guess the process should be to check if debugfs is mounted and the tracing directory exists at the location expected by the application. If yes, we are good to go. Otherwise, mount tracefs at the correct location without disturbing debugfs.
Thoughts?

Reply:
@sanfern  thoughts??

Reply:
tracefs can be mounted in multiple locations. It's good that we don't touch existing mount points in the system, and mount at the location we are looking for.